rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/MatcherMonadSpec.scala

```diff
@@ -80,7 +80,7 @@ class MatcherMonadSpec extends FlatSpec {
     (for {
       _        <- cost.set(Cost(0))
       a        = attemptOpt[F, Unit](failed)
-      res      <- runFirstWithCost(a)
+      res      <- runFirst(a)
```

Comment:
Yet another commit where the wrapper I mentioned would prevent changes in test code :)

My point is twofold:
- repetition in test code is as bad (or worse) as in production code :)
- most of the time it makes sense to extract a `doTheWhen` wrapper in a test :)